includes/fraud-prevention/class-fraud-risk-tools.php

```diff
@@ -82,7 +82,7 @@ public function init_advanced_settings_page() {
 			return;
 		}
 
-		if ( ! $this->payments_account->is_stripe_connected() ) {
+		if ( ! $this->payments_account->is_stripe_connected() || ! $this->payments_account->is_stripe_account_valid() ) {
```

Comment:
As `is_stripe_account_valid` calls `is_stripe_connected`, I think it's safe to just check for `is_stripe_account_valid`, right?

Reply:
Yep, that makes sense. Didn't check the method content before adding it, removed it in [e72e2f3](https://github.com/Automattic/woocommerce-payments/pull/6978/commits/e72e2f368759cded071945691a38378774c7056a).